modules/RTC/ScreenObtainer.js

```diff
@@ -188,8 +117,7 @@ const ScreenObtainer = {
 
             window.JitsiMeetScreenObtainer.openDesktopPicker(
                 {
-                    desktopSharingSources: desktopSharingSources
-                        || this.options.desktopSharingChromeSources
+                    desktopSharingSources
```

Comment:
Since this is now only used in electron, can we hardcode this to `[ 'screen', 'window']` ?

Reply:
in a subsequent PR maybe ? that's what we decided in the standup.